readme.md

```diff
@@ -39,7 +39,7 @@ This repository focuses on examples in Python and only covers common use cases,
 
 ### Get Projects
 
-- The detailed explanation can be accessed [here](https://datasaurai.gitbook.io/datasaur/api/get-data/get-list-of-projects).
+- The detailed explanation can be accessed [here](https://docs.datasaur.ai/api/get-data/get-list-of-projects).
```

Comment:
link di gitbooknya salah seharusnya tanpa "s" https://api-docs.datasaur.ai/#query-getProject
![image](https://github.com/datasaur-ai/api-client-example/assets/8623199/87b3136e-3ed9-4d2f-ad6f-f953503a0a3a)
oiya minor ttg gitbooknya lagi "_You can easily...._" maunya italic tapi ga bener ya

Reply:
ah, link di Gitbook ya, oke
updatenya bukan di repo ini, jadi I'll note to others ya 👍🏼

Reply:
eh, tapi query getProjects ada juga ka Rudi 🤔 
https://api-docs.datasaur.ai/#query-getProjects

getProject untuk retrieve 1

getProjects retrieve list-of-project, based on filter. 

---

anyway, buat italic-nya tetep ku raise ya

Reply:
Oh iya my bad.. tadi aku coba soalnya ga langsung ke section getProjects nya kykny masih loading